lib/policyEvaluator/utils/actionMaps.ts

```diff
@@ -52,6 +52,12 @@ const sharedActionMap = {
     objectPutVersion: 's3:PutObjectVersion',
 };
 
+const actionMapBucketQuotas = {
+    getBucketQuota: 'quota:GetBucketQuota',
+    updateBucketQuota: 'quota:PutBucketQuota',
+    deleteBucketQuota: 'quota:PutBucketQuota',
```

Comment:
I wonder if we should not just consider the APIs as the others, when we deal with specific permissions, like `Put*` when we actually delete it, in such case, I see no need to duplicate the value here, we can just pass the right permission: this also makes it clear in the code what the permission we look for is

Reply:
Done here : https://github.com/scality/Arsenal/pull/2232/commits/b3a5ce5658803f54a2bb32c671d38be2a0d244c0#diff-7fb88a375e077c195e7a6345016d77a135949a6da340c76c6b57c0e3d05cccc9R164